notes/coredns-1.8.4.md

```diff
@@ -0,0 +1,31 @@
++++
+title = "CoreDNS-1.8.4 Release"
+description = "CoreDNS-1.8.4 Release Notes."
+tags = ["Release", "1.8.4", "Notes"]
+release = "1.8.4"
+date = 2021-03-24T07:00:00+00:00
+author = "coredns"
++++
+
+The CoreDNS team has released
+[CoreDNS-1.8.4](https://github.com/coredns/coredns/releases/tag/v1.8.4). This release includes a
+bunch of bugfixes and a few enhancements, see below.
+TODO
+
+## Brought to You By
+
+Chris O'Haver,
+Miek Gieben,
+Mohammad Yosefpor,
+Paco Xu,
+Soumya Ghosh Dastidar.
+
+## Noteworthy Changes
+
+* plugin/metrics: remove RR type (https://github.com/coredns/coredns/pull/4534)
```

Comment:
Maybe be more specific about removal from histograms.

Reply:
Should we bump minor version for changes to existing metrics?